src/GeniusYield/TxBuilder/IO.hs

```diff
@@ -0,0 +1,104 @@
+{-|
+Module      : GeniusYield.TxBuilder.IO
+Copyright   : (c) 2023 GYELD GMBH
+License     : Apache 2.0
+Maintainer  : support@geniusyield.co
+Stability   : develop
+
+-}
+module GeniusYield.TxBuilder.IO (
```

Comment:
This was previously known as `GYTxMonadNode`. Its interface is vastly simplified now. And it is composed with other less powerful monads that are also usable to a library user.

See PR comment about GYTxMonad hierarchy.